external/ILLink/illink.runtimeconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp1.1",
```

Comment:
The new tooling puts this "tfm" in the runtimeconfig.  This value is used by the host to do "shared runtime store" probing.  See [https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/DotNetCore-SharedPackageStore.md](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/DotNetCore-SharedPackageStore.md) for more info.

---
In reply to: [110770450](https://github.com/dotnet/corefx/pull/18035#discussion_r110770450) [](ancestors = 110770450)